resources/views/components/input/label.blade.php

```diff
@@ -1,8 +1,6 @@
-@if (isset($label) && $label)
-    <div class="flex">
-        <span class="text-inactive">@lang($label)</span>
-        @if ($attributes['required'])
-            <span>*</span>
-        @endif
-    </div>
-@endif
+<div class="flex text-inactive">
```

Comment:
why does this need a flex? if you want to get the asterix next to the text, that should be done automatically because the <span element is inline